src/Hook/NodeViewAlterPostRenderHook.php

```diff
@@ -19,11 +19,19 @@ public static function trustedCallbacks(): array {
 
   public static function hook(Markup $markup, array &$output): string
   {
+    $config = \Drupal::config('coyote_img_desc.settings');
     $hostUri = $output['_coyote_node_url'];
+    $disableParsing = $config->get('disable_coyote_filtering');
+    if ($disableParsing) return $markup;
+   
+    $isPublished = $output['#node']->isPublished();
+    $coyote_process_unpublished_nodes = $config->get('coyote_process_unpublished_nodes');
```

Comment:
Can this variable name be renamed as camelCase?

Reply:
> Can this variable name be renamed as camelCase?

yup, will change